### PR TITLE
New landing page

### PR DESCRIPTION
**Pull request by Woodehh**
The current landing page fit in a nice Bootstrap framework. Features:
 - Responsive
 - Metadata complete
 - Opengraph
 - Bootstrap 4.0

### CODE REVIEW

File: layouts/page/single.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
-{{ partial "header.html" . }}
+{{ partial "header-new.html" . }}
 
-<div id="main-wrapper">
-    {{ .Content }}
-</div>
+	<div id="main-wrapper">
```
Comment (metachris) [inline layouts/page/single.html]: don't indent here please

File: static/assets/custom-css/cityofzion.css [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+	font-weight: 600;
+	font-size: 16px;
+	color: white;
+}
+
+/*
+	Main information
+*/
+body {
+	background: #212146 url("../images/website-images/background.jpg") top repeat;
```
Comment (metachris) [inline static/assets/custom-css/cityofzion.css]: use absolute URL here. /assets/...

File: static/assets/css/fa-brands.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+/*!
+ * Font Awesome Free 5.0.8 by @fontawesome - https://fontawesome.com
+ * License - https://fontawesome.com/license (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
+ */
+@font-face {
+  font-family: 'Font Awesome 5 Brands';
+  font-style: normal;
+  font-weight: normal;
+  src: url("../webfonts/fa-brands-400.eot");
```
Comment (metachris) [inline static/assets/css/fa-brands.css]: absolute url's are safer (/assets/...)

File: layouts/partials/header-new.html [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        {{ if .Params.image -}}
+            <link rel="image_src" href="{{ .Site.BaseURL }}/assets/images/opengraph-images/{{ .Params.image }}" />
+            <meta property="og:image" content="{{ .Site.BaseURL }}/assets/images/opengraph-images/{{ .Params.image }}" />
+            <meta property="og:image:url" content="{{ .Site.BaseURL }}/assets/images/opengraph-images/{{ .Params.image }}" />
+            <meta property="og:image:secure_url" content="{{ .Site.BaseURL }}/assets/images/opengraph-images/{{ .Params.image }}" />
+        {{- end }}
+
+        <!-- OpenGraph and Twitter tags -->
+        {{ if .IsPage -}}
+            <meta property="og:type" content="article" />
```
Comment (metachris) [inline layouts/partials/header-new.html]: This `og:type` might conflict with the tag from line 58

File: layouts/partials/header-new.html [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+            <meta property="og:article:published_time" content="{{ .Date.Format "2006-01-02T15:04:05Z07:00" | safeHTML }}" />
```
Comment (metachris) [inline layouts/partials/header-new.html]: Rather not use a hardcoded date. Perhaps we remove that for now

File: layouts/partials/header-new.html [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+	    <meta name="msapplication-square150x150logo" content="/assets/images/icons/mstile-150x150.png" />
+	    <meta name="msapplication-wide310x150logo" content="/assets/images/icons/mstile-310x150.png" />
+	    <meta name="msapplication-square310x310logo" content="/assets/images/icons/mstile-310x310.png" />
+       
+        <!-- OpenGraph Tags -->
+	    <meta property="og:site_name" content="{{ if .Title }}{{ .Title }}{{ else }}{{ .Site.Title }}{{ end }}" />
+	    <meta property="og:title" content="{{ if .Title }}{{ .Title }}{{ else }}{{ .Site.Title }}{{ end }}" />
+	    {{ if .Description }}<meta property="og:description" content="{{ .Description }}" />{{ end }}
+	    <meta property="og:url" content="{{ .Permalink }}" />
+	    <meta property="og:type" content="website" />
```
Comment (metachris) [inline layouts/partials/header-new.html]: remove this, it's covered below

Comment (ITSVision) [reply layouts/partials/header-new.html]: done

File: layouts/partials/header-new.html (hunk 5 above)
Comment (ITSVision) [reply layouts/partials/header-new.html]: done

File: layouts/partials/header-new.html (hunk 4 above)
Comment (ITSVision) [reply layouts/partials/header-new.html]: done

Comment (ITSVision) [reply layouts/partials/header-new.html]: done

File: layouts/page/single.html (hunk 1 above)
Comment (ITSVision) [reply layouts/page/single.html]: done

**Pull request merged**